02.pipeline_demonstration.ipynb

```diff
@@ -0,0 +1,10742 @@
+{
```

Comment:
<p>maybe here we should the 'fresh' clone to be called 'maroon' instead of green, because in the next step it is referred to as maroon</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icenet-ai/icenet-notebooks/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thanks for picking this up Matt! Went through quite a few variations as you can tell :D.</p><p> </p><p>Resolved in new commit.</p><p> </p><p>Be lazy and rename text <code>maroon</code> to <code>green</code>  in outputs.</p>